workshops/views.py

```diff
@@ -1035,3 +1071,16 @@ def _time_series(request, data, title):
     context = {'title': title,
                'data': data}
     return render(request, 'workshops/time_series.html', context)
+
+
+def _failed_to_delete(request, object, back=None):
+    context = {
+        'title': 'Failed to delete',
+        'back': back or object.get_absolute_url,
+        'object': object,
+        'refs': dict(),
+    }
+    for field in dir(object):
+        if field.endswith('_set') and not field.startswith('_'):
```

Comment:
The `_set` check assumes that nobody is setting [`related_name`][1] (more discussion [here][2]).  Even if that assumption was true now it would be brittle, but without #120 we need the current `related_name` for [`Site.organizer`][3].  Is there another way to find all the reverse relationships?

[1]: https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.ForeignKey.related_name
[2]: https://docs.djangoproject.com/en/1.8/topics/db/queries/#related-objects
[3]: https://github.com/swcarpentry/amy/blob/1ab2c6ca628f05b56359d497bf9d0c487ec98822/workshops/models.py#L277